org.unia.gcolor3.json

```diff
@@ -0,0 +1,47 @@
+{
+    "app-id": "org.unia.gcolor3",
+
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+
+    "command": "gcolor3",
+
+    "rename-icon": "gcolor3",
+    "rename-desktop-file": "gcolor3.desktop",
+    "rename-appdata-file": "gcolor3.appdata.xml",
+
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+
+        "--filesystem=xdg-config/gcolor3/config.ini:ro"
```

Comment:
The flatpak is based off v2.2 which has this in the changelog:
> * Follow the XDG directory specification (d91dd06).
> This means that the configuration file has moved from ~/.rgb.ini to $XDG_CONFIG_HOME/gcolor3/config.ini

Or at least I think that was the reasoning, it's been a while since I wrote it

Reply:
Whoops, you're 100% right! I forgot that PR.